plugins/out_es/es.c

```diff
@@ -62,8 +62,8 @@ static flb_sds_t add_aws_auth(struct flb_http_client *c,
     flb_http_add_header(c, "User-Agent", 10, "aws-fluent-bit-plugin", 21);
 
     signature = flb_signv4_do(c, FLB_TRUE, FLB_TRUE, time(NULL),
```

Comment:
So I've heard that:

> You can't include Content-Length as a signed header, otherwise you'll get an invalid signature error.

Currently, content-length is added in the http client `add_host_and_content_length` function which is called when you create a client- do we need to add some code here to ignore that header? I think I remember that when we worked on this before we did...

Reply:
I guess if we test it and we don't get that error response then it works

Reply:
resolved.

I'm wondering if aoss needs `content-type` to also be excluded from the signed headers. This PR does not exclude `content-type` from the signed headers